modules/ROOT/pages/bolt/message.adoc

```diff
@@ -575,6 +579,29 @@ _Servers may choose to include metadata describing the nature of the failure but
 FAILURE {"code": "Example.Failure.Code", "message": "example failure"}
 ----
 
+[[messages-logon]]
+=== Request message `LOGON`
+
+A `LOGON` message carries an authentication request
+
+*Signature:* `6A`
+
+** The scheme is the authentication scheme.
+Predefined schemes are `"none"`, `"basic"`, and `"kerberos"`.
```

Comment:
Scheme values should be without quotes, otherwise it means that the quotes are _part of_ the name. So `none` instead of `"none"`, for example. Applies to later occurrences as well.